sdk/src/main/java/com/uid2/UID2Manager.kt

```diff
@@ -93,7 +95,23 @@ public class UID2Manager internal constructor(
      */
     public var onIdentityChangedListener: UID2ManagerIdentityChangedListener? = null
 
-    private val _state = MutableStateFlow<UID2ManagerState>(NoIdentity)
+    /**
+     * Gets or sets a listener which can be used to determine if the [UID2Manager] instance has finished initializing.
```

Comment:
is a getter for this useful? should we `private get` it?

Reply:
As far as i'm aware, and reminded by Android Studio, is that Kotlin doesn't allow a `getter` to have a different visibility than the field. Basically, you have:
 - Setter visibility must be the same or less permissive than property visibility
 - Getter visibility must be the same as property visibility